src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,7 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+export const UserInfo = ({ user }) => (
```

Comment:
you can also destructure user fields
```suggestion
export const UserInfo = ({{ name, email }}) => (
```